client/ayon_blender/plugins/publish/validate_no_animation.py

```diff
@@ -0,0 +1,45 @@
+
+import bpy
+
+from ayon_core.pipeline.publish import (
+    ValidateContentsOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+import ayon_blender.api.action
+from ayon_blender.api import plugin
+
+
+class ValidateNoAnimation(
+    plugin.BlenderInstancePlugin,
+    OptionalPyblishPluginMixin
+):
+    """Ensure that meshes don't have animation data."""
+
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    families = ["blendScene", "model", "rig"]
```

Comment:
Why does a `blendScene` disallow containing animation? That seems odd? Isn't that a "Allow anything" product type? @LiborBatek ?

Reply:
> I personally dont mind to have such validator for `blendScene` too...just allowing option for more control over content? And still optional too...not an issue imho

Do beware that the default state for it being enabled or active, etc. in settings is not configurable per product type currently - so you may want a model and rig to **require** having no animation (non-optional), but a blend scene (most likely allow it instead) since it's like an "allow me any data".

Reply:
> > I personally dont mind to have such validator for `blendScene` too...just allowing option for more control over content? And still optional too...not an issue imho
> 
> Do beware that the default state for it being enabled or active, etc. in settings is not configurable per product type currently - so you may want a model and rig to **require** having no animation (non-optional), but a blend scene (most likely allow it instead) since it's like an "allow me any data".

@BigRoy 
It is a fair point tbh...but lets keep that for later if someone will argue about it. But didnt think of this scenario which makes very sense (rigs without anim as mandatory by default and non optional vs blendScene having the validator Disabled and/or exposed to user to tweak to his liking)